test/files/rules/norequireimports.test.ts

```diff
@@ -0,0 +1,13 @@
+var lib = require('lib'); // error
+
+let lib2 = require('lib2'); // error
+
+import {l} from 'lib';
+
+var lib3 = load('not_an_import');
+
+var lib4 = lib2.subImport;
+
+var lib5 = require('lib5'),
+    lib6 = require('lib6'),
```

Comment:
these are errors too, right? include `// error` comments consistently through the file